src/pages/WatchLater/WatchLater.jsx

```diff
@@ -1,5 +1,36 @@
 import React from "react";
+import { Navbar, Sidebar, VideoCard } from "components";
+import styles from "./watchlater.module.css";
+import { useVideosOperations } from "context";
 
 export function WatchLater() {
-  return <div>WatchLater</div>;
+  const {
+    videosOperations: { watchLaterVideos },
+  } = useVideosOperations();
+
+  return (
+    <div className={styles.container}>
+      <section className={styles.navbar}>
+        <Navbar />
+      </section>
+      <section className={styles.sidebar}>
+        <Sidebar />
+      </section>
+
+      <main className={styles.main}>
+        <div className={styles.topBar}>
+          {watchLaterVideos.length > 0 ? (
+            <h2 className='text-center text-highlight'>
+              {watchLaterVideos.length} liked videos
+            </h2>
+          ) : null}
```

Comment:
suggestion - you can use `&&` short-circuiting here

Reply:
There is a gotcha with short circuiting conditional rendering more on this here https://stackoverflow.com/questions/53048037/react-showing-0-instead-of-nothing-with-short-circuit-conditional-component .